vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/AbstractGridMultiSelectionModel.java

```diff
@@ -194,14 +207,14 @@ public void updateSelection(Set<T> addedItems, Set<T> removedItems) {
     @Override
     public boolean isSelected(T item) {
         return getSelectedItems().stream().anyMatch(selectedItem -> Objects
-            .equals(getItemId(selectedItem), getItemId(item)));
+                .equals(getItemId(selectedItem), getItemId(item)));
```

Comment:
You need to fix your code formatting settings.  Import the code settings in https://github.com/vaadin/flow/tree/master/eclipse then reformat this file.

Reply:
You can set automatic formatting rules so your editor will automatically add spaces etc. where they should be. Here are the instructions: 
https://docs.google.com/document/d/1_1S2VILmr8y5B3hD_fMbb9SwpkFKZjnAmaW2oE4_uYo/edit#heading=h.c19b3mugjara

Reply:
fixed it back.